src/Compilers/Core/Portable/InternalUtilities/ReferenceEqualityComparer.cs

```diff
@@ -23,6 +23,11 @@ bool IEqualityComparer<object>.Equals(object a, object b)
 
         int IEqualityComparer<object>.GetHashCode(object a)
         {
+            return ReferenceEqualityComparer.GetHashCode(a);
```

Comment:
just did not want to duplicate that call. to make sure the implementations are exactly the same.